src/donutty.js

```diff
@@ -33,6 +33,18 @@
 
         }
 
+        if ( !this.$wrapper ) {
+
+            return this;
+
+        }
+
+        if (options === null) {
+
+            options = this.getOptionsFromTag();
```

Comment:
```suggestion
            options = JSON.parse( JSON.stringify( this.$wrapper.dataset ) );
```
Have to move this, as the declaration for `this.$wrapper` previously ran in `init()` which would not be available at runtime